views/randomBeer.hbs

```diff
@@ -0,0 +1,10 @@
+ <div>
+      {{#each randomBeers}}
+    <img src='{{image_url}}' alt='{{name}}' />
+      <h4>{{name}}</h4>
+      <h5>{{tagline}}</h5>
+      <p>{{description}}</p>
+      <p>{{food_pairing}}</p>
```

Comment:
if you take a look on your api response, you will see that food_pairing is an array of strings. 
So, to render it as a list you need to make another loop and print each one of the items, like this:

```
<div>
  {{#each randomBeers}}
    <img src='{{image_url}}' alt='{{name}}' />
    <h4>{{name}}</h4>
    <h5>{{tagline}}</h5>
    <p>{{description}}</p>
    <ul>
      {{#each food_pairing}}
        <li>{{this}}</li>
      {{/each}}
    </ul>
    <p>{{brewers_tips}}</p>
  {{/each}}
</div>
```